packages/application-shell/src/components/authenticated/am-i-logged-in.tsx

```diff
@@ -1,11 +1,12 @@
+import type { TQueryVariablesWithTarget } from '@commercetools-frontend/constants';
 import type {
   TAmILoggedInQuery,
   TAmILoggedInQueryVariables,
 } from '../../types/generated/mc';
 
 import React from 'react';
-import { useQuery } from 'react-apollo';
 import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
+import { useMcQuery } from '../../hooks/apollo-hooks';
```

Comment:
These new hooks are just a wrapper around the original apollo hooks. The only difference is that the `context` object is now properly typed instead of being `any`.